_data/simple-icons.json

```
@@ -1567,6 +1567,11 @@
             "hex": "CB2E6D",
             "source": "https://azure.microsoft.com/products/devops"
         },
+        {
+            "title": "Azure Blobs",
+            "hex": "0072C6",
+            "source": "https://azure.microsoft.com/content/dam/microsoft/legacy/acom/svg/h-z/storage-blobs.svg"
```

```suggestion
            "source": "https://learn.microsoft.com/azure/architecture/icons"
```

If the icon is available here, we should probably use this source. A lot of our other Azure icons come from there.